app/mutations/product/createProductMutation.ts

```diff
@@ -0,0 +1,31 @@
+import {graphql} from 'react-relay';
+import {
+  createProductMutation as createProductMutationType,
+  createProductMutationVariables
+} from 'createProductMutation.graphql';
+import {RelayModernEnvironment} from 'relay-runtime/lib/store/RelayModernEnvironment';
+import BaseMutation from '../BaseMutation';
+
+const mutation = graphql`
+  mutation createProductMutation($input: CreateProductInput!) {
+    createProduct(input: $input) {
```

Comment:
Same thing here. These mutations were created way before we started adding the clientMutationId
```suggestion
     clientMutationId
    createProduct(input: $input) {
```

Reply:
Added